src/main/java/duke/Event.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Contains additional information relevant to an Event task.
+ * 
+ * @author Siau Wee
+ */
+public class Event extends Task {
+
+    private final DateTimeFormatter FORMAT_MMDDYY_TIME = 
+            DateTimeFormatter.ofPattern("MM/dd/yy, HH:mm");
+            
+    private LocalDateTime eventTime;
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMAT_MMDDYY_TIME = 
            DateTimeFormatter.ofPattern("MM/dd/yy, HH:mm");

    private final LocalDateTime eventTime;
```

Suggesting the addition of a few modifiers.